packages/transformers/html/src/HTMLTransformer.js

```diff
@@ -11,13 +11,13 @@ import extractInlineAssets from './inline';
 
 export default new Transformer({
   canReuseAST({ast}) {
-    return ast.type === 'posthtml' && semver.satisfies(ast.version, '^0.4.0');
+    return ast.type === 'posthtml' && semver.satisfies(ast.version, '^0.11.3');
```

Comment:
The version used before was the `parser` version, but when we parse it gives us back the version of `posthtml`, so I am updating these to this.

Reply:
does the parser result actually include version information? Last I remember the posthtml ast is basically unversioned (I don't think they've made any breaking changes so far either).

Personally it doesn't matter to me as long as we commit to something and make sure third party plugins follow it.

Reply:
I looked into it and I didn't see a version returned from `posthtml` or the `parser` so I reverted back to using the parser version, as it felt more right.